pom.xml

```diff
@@ -68,6 +68,7 @@
         <aws.s3.version>[1.12.62,1.13.0)</aws.s3.version>
         <junit.version>4.13.2</junit.version>
         <logback.version>1.2.11</logback.version>
+        <logback-slf4j-v2.version>1.4.4</logback-slf4j-v2.version>
```

Comment:
This slf4j-api v2 compatible version is supposed to be used only for test code and example projects. See also: https://github.com/slackapi/java-slack-sdk/issues/1034